packages/devtools/lib/src/main.dart

```diff
@@ -132,71 +130,94 @@ class PerfToolFramework extends Framework {
 
   Future<bool> isFlutterApp() async {
     final EvalOnDartLibrary flutterLibrary = EvalOnDartLibrary(
-      flutterLibraryUriCandidates,
+      [flutterLibraryUri, flutterWebLibraryUri],
       serviceManager.service,
     );
 
     try {
-      await flutterLibrary.libraryRef.catchError(
-        (_) => throw FlutterLibraryNotFound(),
-        test: (e) => e is LibraryNotFound,
-      );
-    } catch (e) {
-      if (e is FlutterLibraryNotFound) {
-        return false;
-      }
+      await flutterLibrary.libraryRef;
+    } on LibraryNotFound catch (_) {
+      return false;
+    }
+    return true;
+  }
+
+  Future<bool> isFlutterWebApp() async {
+    // TODO(kenzie): fix this if screens should still be disabled when flutter
+    // merges with flutter_web.
+    final EvalOnDartLibrary flutterWebLibrary = EvalOnDartLibrary(
+      [flutterWebLibraryUri],
+      serviceManager.service,
+    );
+
+    try {
+      await flutterWebLibrary.libraryRef;
+    } on LibraryNotFound catch (_) {
+      return false;
     }
     return true;
   }
 
   Future<bool> isProfileBuild() async {
     try {
       final Isolate isolate = await serviceManager.service
-          .getIsolate(serviceManager.isolateManager.selectedIsolate.id);
+          .getIsolate(serviceManager.isolateManager.isolates.first.id);
+      // This evaluate statement will throw an error in a profile build.
       await serviceManager.service.evaluate(
-        serviceManager.isolateManager.selectedIsolate.id,
+        serviceManager.isolateManager.isolates.first.id,
         isolate.rootLib.id,
         '1+1',
       );
       // If we reach this return statement, no error was thrown and this is not
       // a profile build.
       return false;
-    } catch (e) {
-      if (e is RPCError) {
-        return true;
-      }
+    } on RPCError catch (_) {
+      return true;
     }
-    return false;
   }
 
   Future<void> addScreens() async {
     final _isFlutterApp = await isFlutterApp();
+    final _isFlutterWebApp = await isFlutterWebApp();
     final _isProfileBuild = await isProfileBuild();
 
     addScreen(InspectorScreen(
-      disabled: !_isFlutterApp || _isProfileBuild,
-      disabledTooltip: !_isFlutterApp
-          ? 'This section is disabled because you are not running a Flutter '
+      disabled: !_isFlutterApp || !_isFlutterWebApp || _isProfileBuild,
```

Comment:
add an _isAnyFlutter (pick a better name)
that is _isFlutterApp || _isFlutterWebApp
I see that case in a bunch of places
this case would then be
disabled: !_isAnyFlutterApp || isProfileBuild

Reply:
done.